.env.example

```
@@ -56,15 +56,16 @@ OBSERVABILITY_SERVER_BASE_PATH=/api
+OBSERVABILITY_SERVER_TLS_CERTFILE=cert.pem
+OBSERVABILITY_SERVER_TLS_KEYFILE=key.pem
+OBSERVABILITY_SERVER_ENABLE_TLS=true
 
 # Inter-service URLs
-SERVICES_GATEWAY_URL=http://localhost:8080/api
-SERVICES_TENANT_URL=http://localhost:8082/api
-SERVICES_GUARDRAIL_URL=http://localhost:8083/api
-SERVICES_OBSERVABILITY_URL=http://localhost:8084/api
+SERVICES_GATEWAY_URL=https://localhost:8080/api
+SERVICES_TENANT_URL=https://localhost:8082/api
+SERVICES_GUARDRAIL_URL=https://localhost:8083/api
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SERVICES_GUARDRAIL_URL key should go before the SERVICES_TENANT_URL key

(UnorderedKey)